### PR TITLE
Added lights turning back to a default color automatically

### PR DESCRIPTION
**Pull request by jurrejelle**

### CODE REVIEW

File: src/main/java/com/HuePlugin/Fireworks.java [hunk 1]
```diff
@@ -38,8 +38,12 @@ public void run() {
 
             }
         }
+        
+		this.room.setState(State.builder().color(Color.of(config.defaultColorConfig())).on());
```
Comment (Nick-Anzalone) [inline src/main/java/com/HuePlugin/Fireworks.java]: You can not retrieve information from the config in this file. 

Maybe when creating the thread, pass this default color as an argument.

File: src/main/java/com/HuePlugin/HuePlugin.java [hunk 2]
```diff
@@ -106,12 +114,20 @@ public void onChatMessage(ChatMessage chatMessage)
 			if(chatMessage.getMessage().equals("!Testhueconnection")){
 				if(connectionSucessful){
 					client.addChatMessage(ChatMessageType.GAMEMESSAGE,"","Your smart lights are connected.","");
-
 				}else{
 					client.addChatMessage(ChatMessageType.GAMEMESSAGE,"","Your smart lights are not connected.","");
-
 				}
+			}
+			if(chatMessage.getMessage().equals("!Testlights")) {
```
Comment (Nick-Anzalone) [inline src/main/java/com/HuePlugin/HuePlugin.java:121]: With this addition you can remove the "!Testhueconnection" feature and have this as the sole feature to confirm your connection has been established.

**Review by Nick-Anzalone — CHANGES_REQUESTED** [review]
See comments in order to resolve an issue I had noticed in the Firework.java file.

I also think that with the "!Testlights" feature you can remove the "!Testhueconnection" feature. 

The rest looks good though!

File: src/main/java/com/HuePlugin/Fireworks.java (hunk 1 above)
Comment (jurrejelle) [reply src/main/java/com/HuePlugin/Fireworks.java]: Yup, that's on me. Changed!

File: src/main/java/com/HuePlugin/HuePlugin.java (hunk 2 above)
Comment (jurrejelle) [reply src/main/java/com/HuePlugin/HuePlugin.java:121]: Yup, changed! Sounds good!

**Pull request merged**